test/e2e/framework.go

```diff
@@ -2863,7 +2863,7 @@ func (data *TestData) createDaemonSet(name string, ns string, ctrName string, im
 	podSpec := corev1.PodSpec{
 		Tolerations: controlPlaneNoScheduleTolerations(),
 		// Set it to 1s for immediate shutdown to reduce test run time and to avoid affecting subsequent tests.
-		TerminationGracePeriodSeconds: pointer.Int64(1),
+		TerminationGracePeriodSeconds: ptr.To(int64(1)),
```

Comment:
Looks like the correct way should be `ptr.To[int64](1)`, please check other occurrences.

Reply:
Fix, change other usages to `ptr.To[Type]` unless static case is necessary.